examples/scripts/blender/sdf_exporter.py

```diff
@@ -46,21 +46,22 @@ def export_sdf(prefix_path):
         geometry = ET.SubElement(visual, "geometry")
         mesh = ET.SubElement(geometry, "mesh")
         uri = ET.SubElement(mesh, "uri")
-        uri.text = dae_filename
+        uri.text = os.path.join(meshes_folder_prefix, dae_filename)
```

Comment:
While we're here, could you apply the same fix for line 76 `albedo_map.text`, line 81 `light_map.text`, and line 143 `uri.text` for the collision? Right now, the visual tag exports `meshes/model.dae`, but the collision tag still just says `model.dae`.

Reply:
Thank you! I missed that one. Fixed in https://github.com/gazebosim/gz-sim/pull/1718/commits/f8dc7631639b392fe043450dec7caa4fd4c74209

This is the SDF that gets generated now:
```xml
<?xml version="1.0" ?>
<sdf version="1.8">
  <model name="test">
    <link name="testlink">
      <visual name="LPCoke_Cube.000">
        <geometry>
          <mesh>
            <uri>meshes/model.dae</uri>
            <submesh>
              <name>LPCoke_Cube.000</name>
            </submesh>
          </mesh>
        </geometry>
        <material>
          <diffuse>1.0 1.0 1.0 1.0</diffuse>
          <specular>0.0 0.0 0.0 1.0</specular>
          <pbr>
            <metal>
              <albedo_map>meshes/coke.png</albedo_map>
            </metal>
          </pbr>
        </material>
      </visual>
      <collision name="collision">
        <geometry>
          <mesh>
            <uri>meshes/model.dae</uri>
          </mesh>
        </geometry>
        <surface/>
        <contact/>
        <collide_bitmask>0x01</collide_bitmask>
      </collision>
    </link>
  </model>
  <static>true</static>
</sdf>
```